src/main/java/com/epam/test_generator/controllers/caze/CaseDTOsTransformer.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.test_generator.controllers.caze;
+
+import com.epam.test_generator.controllers.caze.request.CreateCaseDTO;
+import com.epam.test_generator.controllers.caze.response.CaseDTO;
+import com.epam.test_generator.entities.Case;
+import com.epam.test_generator.transformers.StepTransformer;
+import com.epam.test_generator.transformers.TagTransformer;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class CaseDTOsTransformer{
+
+    public List<CaseDTO> toDtoList(List<Case> cases) {
+        return cases.stream().map(this::toDto).collect(Collectors.toList());
+    }
+
+    public CaseDTO toDto(Case caze) {
+        CaseDTO caseDTO = new CaseDTO();
+        caseDTO.setName(caze.getName());
+        caseDTO.setDescription(caze.getDescription());
+        caseDTO.setSteps(new StepTransformer().toDtoList(caze.getSteps()));
+        caseDTO.setCreationDate(caze.getCreationDate());
+        caseDTO.setUpdateDate(caze.getUpdateDate());
+        caseDTO.setPriority(caze.getPriority());
+        caseDTO.setTags(caze.getTags().stream().map(new TagTransformer()::toDto).collect(Collectors.toSet()));
+        caseDTO.setStatus(caze.getStatus());
+        caseDTO.setComment(caze.getComment());
+        caseDTO.setJiraProjectKey(caze.getJiraProjectKey());
```

Comment:
Do we need jira related stuff on UI?

Reply:
I forgot - did we decide not to return anything related to Jira to UI?

Reply:
Hm... Let's remove it for now since UI is not using it yet.

Reply:
I forgot - did we decide not to send any Jira-related info to UI?

Reply:
Ok. I removed all Jira-related fields from response.CaseDTO